js/modules/swiperHelpers.js

```diff
@@ -0,0 +1,58 @@
+export function setSlidesState(){
+	const swiper = this;
+	const activeIndexes = getActiveIndexes.call(swiper);
+	const slideCollection = swiper.slides;
+	const mqlOver1920 = window.matchMedia("(min-width: 1920px)");
+
+	if(mqlOver1920.matches){
+		slideCollection.forEach(function(elem, index) {
+			if(activeIndexes.includes(index)){
+				elem.classList.remove("activeSlides");
+			}
+			else{
+				elem.classList.add("activeSlides");
+			}
+		});
+	}
+
+	mqlOver1920.addEventListener("change", on1920changeStateHandler);
```

Comment:
you use `setSlidesState` for slideChange event, it means that every time a slide is changed you set a handler.
Do you really think it is okay?

Reply:
Use such a code in more specific files like `swiperOurHorsesFunctionality.js` but not in helpers

Reply:
It's used in both swipers. It's general functionality.

Reply:
I don't agree with you again. That "change" event doesn't concern swiper. It concerns "mqlOver1920", it's MediaQueryList.

Reply:
function `setSlidesState` is called every time slideChange event fires.
in the code of this function you set the hanlder for mqlOver1920
```js
mqlOver1920.addEventListener("change", on1920changeStateHandler);
```
Do you really think it is okay?

Reply:
Done.